<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZX6R</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow { width: 318px; height: 238px; }

#centralwidget { background-color: black; color: white; }

QWidget { background-color: #262424; color: white; font-size: 12px; }

QPushButton { background-color: black; color: white; border: 1px solid #4A4646; }

QTabBar::tab { min-width: 56px; font-weight: normal; color: #DC3434; padding: 10px 22px; border: 0; background-color: black; }

QTabBar::tab:selected { font-weight: bold; background-color: #4A585D; color: #85F187; }

QTabWidget::pane { border: 2px solid red; border-radius: 7px; background-color: #3E3E3E;}

#labelBestLapTime, #labelLastLapTime { font-size: 40px; font-weight: bold; }

#labelBestLapTime {color: #24A523; }
#labelLastLapTime {color: #9098A9; }

#tabStatus QLabel { padding: 4px 2px;  qproperty-alignment: AlignCenter; background-color: black; font-size: 11px; }

#pbRecord {  border-style: solid; border-width: 3px; border-color: #585858; border-radius: 8px; }

#pbLiveStatus { font-size: 20px; font-weight: bold; border-radius: 8px; background-color: #463535; color: #AB0A0A; }
#pbLiveStatus:checked { background-color: #164321; color: #1B5028; }

#pbShutdown, #pbReboot { border-radius: 4px; color: #D5A6A6; }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabApplication">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <widget class="QWidget" name="tabMain">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <widget class="QPushButton" name="pbRecord">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>271</width>
          <height>41</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Record</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelLastLapTime">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>241</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>03:88:456</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="labelBestLap">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>140</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Best:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="labelLastLap">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Last:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="labelBestLapTime">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>130</y>
          <width>241</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>03:88:456</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabStatus">
       <attribute name="title">
        <string>Status</string>
       </attribute>
       <widget class="QLabel" name="labelStatusGPS">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>71</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>GPS</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelStatusGyroscope">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>70</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Lean</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelStatusAccelerometer">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>70</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>GForce</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelStatusKDS">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>120</y>
          <width>70</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>KDS</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelStatus_lat">
        <property name="geometry">
         <rect>
          <x>71</x>
          <y>0</y>
          <width>81</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>lat</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelStatus_lon">
        <property name="geometry">
         <rect>
          <x>152</x>
          <y>0</y>
          <width>81</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>lon</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelStatus_gyros">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>223</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>degrees</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelStatus_accelerometer">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>90</y>
          <width>223</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>raw</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelStatus_rpm">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>120</y>
          <width>101</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>rpm</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelStatus_gear">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>120</y>
          <width>43</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>gear</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelStatusCompass">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>70</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Compass</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelStatus_heading">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>223</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>heading</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelStatus_speed">
        <property name="geometry">
         <rect>
          <x>232</x>
          <y>0</y>
          <width>61</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>kph</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbReboot">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>148</y>
          <width>63</width>
          <height>31</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Reboot</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbShutdown">
        <property name="geometry">
         <rect>
          <x>2</x>
          <y>148</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Shutdown</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbLiveStatus">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>148</y>
          <width>144</width>
          <height>31</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>LIVE</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="labelStatus_kph">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>120</y>
          <width>81</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>kph</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabLaps">
       <attribute name="title">
        <string>Tools</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
